experiments/ClimaEarth/test/component_model_tests/climaland_tests.jl

```diff
@@ -1,11 +1,17 @@
 using Test
 import ClimaCore as CC
+import ClimaAtmos as CA
 import ClimaCoupler
+import ClimaCoupler: FluxCalculator, Interfacer
 import Dates
+import ClimaComms
+@static pkgversion(ClimaComms) >= v"0.6" && ClimaComms.@import_required_backends
```

Comment:
Would it make sense to move this to the compat and then remove from here?

Reply:
This line is like this so we're still compatible with previous versions of ClimaComms, which could be helpful if we ever need to debug/test with a previous version